### ISSUE TITLE
Error when viewing records with many holdings

### ISSUE THREAD

**Issue by jenniferward** [rism-digital/muscat#1619]
@MFalletta and I have encountered this error when viewing records that have dozens of holdings:
![image](https://github.com/user-attachments/assets/9f891761-b74a-437f-8e01-579d4c1035a0)

In German it looks like this: 
![image (2)](https://github.com/user-attachments/assets/6a48965c-0b9b-424b-a0fd-6e62b1efee8d)

It is happening right now with https://muscat.rism.info/admin/sources/1001299207 and  https://muscat.rism.info/admin/sources/852028308 -- Windows 10, Chrome. These are both treatises.

I think https://muscat.rism.info/admin/sources/990053593 holds the record for most holdings, but it is not triggering the error right now.

**Comment by jenniferward** [comment]
Yes it is happening right now with https://muscat.rism.info/admin/sources/852028308! 
![image](https://github.com/user-attachments/assets/b953ec7c-5b60-4f13-951c-aa18d9699ad9)

**Comment by xhero** [comment]
How often does it happen? I can't seem to reproduce it, not even if I storm muscat with refreshes...

**Comment by ahankinson** [comment]
When it happens open up the web inspector in Chrome and take a screenshot of any errors in the console. 403 would suggest that some credentials are not being sent.. maybe a CORS or some cross-origin problem where a cookie can’t be sent to a different url?

**Comment by xhero** [comment]
I wonder if it is the SBB proxy blowing up, I can't find 403s in the logs in muscat and depending from the connection I'm using (eduroam, RISM wifi or VPN) I get this error always (eduroam) or never (VPN)

**Comment by jenniferward** [comment]
Like this? https://muscat.rism.info/admin/sources/852028308

![image](https://github.com/user-attachments/assets/442702aa-54b9-4e56-a024-65d8e8a0f1c4)

**Comment by jenniferward** [comment]
Might be similar to the error reported here: https://github.com/rism-digital/muscat/issues/1461

**Comment by jenniferward** [comment]
Got a different error just now with https://muscat.rism.info/admin/sources/990042920

![Image](https://github.com/user-attachments/assets/c7dda43a-2f5e-48a0-96a7-902339d0dabc)

![Image](https://github.com/user-attachments/assets/2935c969-e995-4627-9f7b-6cbc0aecce07)

**Comment by xhero** [comment]
Fixed in aae58ea65e99d0c75411e3304ab25941189fda87, the holdings are now generated in the backend.
